roles/ansible-test-splitter/tasks/main.yaml

```diff
@@ -1,31 +1,60 @@
 ---
-- copy:
-    src: list_changed_targets.py
-    dest: /tmp/list_changed_targets.py
-    mode: '0700'
-
-- set_fact:
-    splitter_cmd: >
-      python3 /tmp/list_changed_targets.py
-      --branch {{ zuul.branch }}
-      {% if ansible_test_splitter__releases_to_test is defined %}--ansible-releases {{ ansible_test_splitter__releases_to_test | join(' ') }}{% endif %}
-      {% if ansible_test_splitter__total_job is defined %}--total-job {{ ansible_test_splitter__total_job }}{% endif %}
-      {% if ansible_test_splitter__test_changed|bool %}--test-changed{% else %}--test-all-the-targets{% endif %}
-      {{ ansible_test_splitter__check_for_changes_in | join(' ') }}
-
-- name: Will split up the jobs with the following command
-  debug:
-    msg: "{{ splitter_cmd }}"
-
-- name: Split up the workload
-  command: "{{ splitter_cmd }}"
-  register: _result
-
-- set_fact:
-    for_zuul_return: '{{ _result.stdout | from_json }}'
-
-- debug: var=for_zuul_return
-
-- name: Return the result to Zuul
-  zuul_return:
-    data: "{{ for_zuul_return.data }}"
+- block:
+    - name: Create temporary directory for tests
+      tempfile:
+        state: directory
+        suffix: .splitter
+      register: _result
+
+    - set_fact:
```

Comment:
I don't think we need this, we've got already the `json` and `requests` packages in the container that we use to run the job:

```
⋊> ~ podman run quay.io/software-factory/zuul-worker-ansible python -c 'import json;print(json.__version__)'                                                                                                                          10:48:12
2.0.9
⋊> ~ podman run quay.io/software-factory/zuul-worker-ansible python -c 'import requests;print(requests.__version__)'                                                                                                                  10:48:19
2.27.1
```